app/controllers/application_controller.rb

```diff
@@ -6,16 +6,17 @@ class ApplicationController < ActionController::Base
   def append_info_to_payload(payload)
     super
     payload[:host] = request.host
-    case payload[:status]
-    when 200
-      payload[:level] = "INFO"
-    when payload[:status]  > 200 && payload[:status]  < 400
-      payload[:level] = "DEBUG"
-    when payload[:status]  > 400
-      payload[:level] = "WARNING"
-    when payload[:status]  >= 500
-      payload[:level] = "ERROR"
-    end
+    #console.log('payload[:host]')
+    #case payload[:status]
+    #when 200
+    #  payload[:level] = "INFO"
+    #when payload[:status]  > 200 && payload[:status]  < 400
+    #  payload[:level] = "DEBUG"
+    #when payload[:status]  > 400
+    #  payload[:level] = "WARNING"
+    #when payload[:status]  >= 500
+    #  payload[:level] = "ERROR"
+    #end
```

Comment:
These lines are commented out because the payload object is null right now.  I'm hoping to fix in the next iteration.